runtime/onert/api/python/package/experimental/train/session.py

```diff
@@ -1,304 +1,276 @@
+import time
 import numpy as np
+from typing import Any, List, Tuple, Dict, Union, Optional
 
 from onert.native import libnnfw_api_pybind
-from onert.native.libnnfw_api_pybind import optimizer as optimizer_type
-from onert.native.libnnfw_api_pybind import loss as loss_type
+# from onert.native.libnnfw_api_pybind import optimizer as optimizer_type, loss as loss_type
```

Comment:
Maybe.. forgot to remove the commented line? 

```suggestion
```

Reply:
I removed it.